test/functionalTest/cases/3272_custom_notification_sending_0-length_request/custom_notification_sending_0-length_request.test

```diff
@@ -0,0 +1,567 @@
+# Copyright 2020 Telefonica Investigacion y Desarrollo, S.A.U
```

Comment:
I'd suggest to split this test in two .test, one focused in the subscriptions API and another in notifications.

The one focused in subscriptions API could be as follows:

```
# 01. Create sub1 with payload="abc", includePayload=false
# 02. Create sub2 with payload="", includePayload=false
# 03. Create sub3 without payload, includePayload=false
# 04. Create sub4 with payload="abc", includePayload=true
# 05. Create sub5 with payload="", includePayload=true
# 06. Create sub6 without payload, includePayload=true
# 07. Create sub7 with payload="abc", without includePayload
# 08. Create sub8 with payload="", without includePayload
# 09. Create sub9 without payload, without includePayload
# 10. GET /v2/subscription and see the 9 subscriptions
```

The one focused in notifications could be as follows:

```
01. Create sub1 with payload="abc", includePayload=false
02. Create sub2 without payload, includePayload=false
03. Create entity E1/T1
04. Dumps acummulator (see 2 notifications, both without payload)
```

Reply:
With regards the first case (subscription API test) please take into account the considerations already described in https://github.com/telefonicaid/fiware-orion/pull/3683/files#r501631288

Reply:
Added test case